pkg/kubelet/kubelet.go

```diff
@@ -2112,6 +2132,25 @@ func (kl *Kubelet) syncLoopIteration(updates <-chan kubetypes.PodUpdate, handler
 			// TODO: Do we want to support this?
 			glog.Errorf("Kubelet does not support snapshot update")
 		}
+	case e := <-plegCh:
```

Comment:
We should drain the `plegCh` during every `syncLoopIteration` right? IIUC, this logic processes only one event as of now.